src/app/shared/services/api/archive.repo.ts

```diff
@@ -153,6 +153,10 @@ export class ArchiveRepo extends BaseRepo {
 
     return this.http.sendRequestPromise<ArchiveResponse>('/profile_item/delete', data, ArchiveResponse);
   }
+
+  public getArchiveTags(archiveId: string) {
+    return this.httpV2.get<ArchiveResponse>(`v2/archive/${archiveId}/tags/public`).toPromise();
```

Comment:
```suggestion
    return this.httpV2.get<TagVOData>(`v2/archive/${archiveId}/tags/public`).toPromise();
```
The typing here is inaccurate; according to the [API documentation](https://github.com/PermanentOrg/stela/blob/main/API.md#archives) this endpoint currently returns an array of TagVOs. (To get this fix to work you'll also have to import the `TagVOData` type as well!)

Reply:
@meisekimiu thanks!